tools/jenkins/Jenkinsfile-api

```diff
@@ -0,0 +1,107 @@
+pipeline{
```

Comment:
Do we need jenkins file for openshift 3?

Reply:
Let me delete the file.